src/panels/GettingStartedPanel/Steps/IntroductionSteps.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Image, List, Stack, Text } from '@mantine/core';
+import { Alert, Image, List, Stack, Text } from '@mantine/core';
+
+import { useAppSelector } from '@/redux/hooks';
 
 import { TaskCheckbox } from '../Tasks/Components/TaskCheckbox';
 
 export function useIntroductionSteps(): React.ReactNode[] {
+  const profileName = useAppSelector((state) => state.profile.name);
   const { t } = useTranslation('panel-gettingstartedtour', {
     keyPrefix: 'steps.introduction'
   });
+  const isDefaultProfile = profileName === 'Default';
```

Comment:
this is not going to catch if a user profile is called default

Reply:
Also random find but apparently a lot of our profiles names are "Default"... making an issue